chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/report/attestation/AttestationReportDAO.java

```diff
@@ -43,13 +73,26 @@ public List<AttestationReport> getAttestationReportByAttestationPeriod(Attestati
                 .toList();
     }
 
+    public List<AttestationReportDeveloper> getAttestationReportDeveloperByAttestationPeriod(AttestationPeriod period) {
+        return getAttestationReportDeveloperEntitiesByAttestationPeriod(period).stream()
+                .map(entity -> entity.toDomain())
+                .toList();
+    }
+
     public void deleteAttestationReportByDate(LocalDate date) {
         getEntitiesByDate(date).forEach(entity -> {
             entity.setDeleted(true);
             update(entity);
         });
     }
 
+    public void deleteAttestationReportDeveloperByAttestationReportId(Long attestationReportId) {
+        Query query = entityManager.createQuery(
+                "delete from AttestationReportDeveloperEntity where attestationReport.id = :attestationReportId");
```

Comment:
The above method (not part of the diff) looks like we are using a soft delete. This method looks like a regular delete. Are they different intentionally or should they match?

Reply:
Method is not used - I removed it.